testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -321,14 +333,38 @@ impl BlockMinerThread {
         for (_slot, signer_message) in signer_messages {
             match signer_message {
                 SignerMessage::Transactions(transactions) => {
-                    // TODO: filter out transactons that are not valid and that do not come from the signers
-                    // TODO: move this filter function from stacks-signer and make it globally available perhaps?
-                    transactions_to_include.extend(transactions);
+                    for transaction in transactions {
```

Comment:
We cannot do this. A signer might have crashed and not updated their stackerdb. And a malicious signer can add whatever they want to a stackerdb instance. It is not externally controllable. Other signers already filter during their check on the block. But we cannot trust signers to behave as expected. I do agree that this functionality should be gloabally available though instead of having htese two sep functions. Should have a util function, but this is a placeholder for now.